homeassistant/components/media_player/roku.py

```diff
@@ -1,79 +1,49 @@
 """
-Support for the roku media player.
+Support for the Roku media player.
 
 For more details about this platform, please refer to the documentation at
 https://home-assistant.io/components/media_player.roku/
 """
 import logging
 
-import voluptuous as vol
-
+from homeassistant.components.roku import (DATA_ENTITIES)
 from homeassistant.components.media_player import (
-    MEDIA_TYPE_MOVIE, PLATFORM_SCHEMA, SUPPORT_NEXT_TRACK, SUPPORT_PLAY,
+    MEDIA_TYPE_MOVIE, SUPPORT_NEXT_TRACK, SUPPORT_PLAY,
     SUPPORT_PLAY_MEDIA, SUPPORT_PREVIOUS_TRACK, SUPPORT_SELECT_SOURCE,
     SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_SET, MediaPlayerDevice)
 from homeassistant.const import (
     CONF_HOST, STATE_HOME, STATE_IDLE, STATE_PLAYING, STATE_UNKNOWN)
-import homeassistant.helpers.config_validation as cv
 
 REQUIREMENTS = ['python-roku==3.1.5']
```

Comment:
This should be removed, this is a requirement of the component that this platform is dependant on.

Reply:
Good catch! Fixed!